README.md

```diff
@@ -26,7 +26,7 @@ we are gathered here today to get through this thing called LIFE."</em><strong>
   </div>
    </br>
 
-# e-wallet app
+# Budget app
```

Comment:
Kindly deploy the application using Heroku, Netlify, or GitHub Pages, and add the link to the deployed project in the README 👍

Reply:
Done.